README.md

```diff
@@ -12,33 +12,33 @@ In the example below, a `score.yaml` file is executed via [score-compose](https:
 
 This project aims to reduce developer toil and cognitive load by only having to define a single yaml file that works across multiple platforms.
 
-Score helps to improve the developer experience by being combined with other workflows and is not meant to replace yaml. Infrastructure and development teams can use this specification to improve workflows throughout their organization.
+Score helps to improve the developer experience by being combined with other workflows, and is not meant to replace yaml. Infrastructure and development teams can use this specification to improve workflows throughout their organization.
 
 Using Score provides the following advantages:
 
 - The developer only needs to learn the _Score Specification_ to define the workloads that run across all these platforms.
 - The _Score Specification_ is designed to be applied to multiple environments to reduce the amount of configuration.
 - An infrastructure team can help define and configure the resources and multiple platforms where the developer can work on.
 
-At this time Score is still in alpha, but we are hoping for it to quickly grow its implementation use cases through community contributions.
+At this time, Score is still in alpha, but we are hoping for it to quickly grow its implementation use cases through community contributions.
 
 ## ![Score](/docs/images/logo.svg) Why Score?
 
-Cloud native developers often struggle with configuration inconsistencies between environments. This gets even more complicated when the technology stack in each environment is different. What if you use Docker Compose for local development, but Helm Charts to deploy to the Kubernetes based development environment?
+Cloud native developers often struggle with configuration inconsistencies between environments. This gets even more complicated when the technology stack in each environment is different. What if you use Docker Compose for local development, but Helm Charts to deploy to the Kubernetes-based development environment?
 
 Not only do you have to figure out Docker Compose and Helm, but you need to keep them in sync!
 
 This results in various bottlenecks along the application delivery lifecycle.
 
-  🎵 Tech & tools that require specialized knowledge and operational expertise are imposed on developers.
+🎵 Tech & tools that require specialized knowledge and operational expertise are imposed on developers.
 
-  🎵 Different configuration rules, constructs and values between local and remote environments increase the risk of configuration inconsistencies.
+🎵 Different configuration rules, constructs and values between local and remote environments increase the risk of configuration inconsistencies.
 
-  🎵 Keeping a multitude of platform- and environment-specific configuration files in sync leads to repetitive configuration work.
+🎵 Keeping a multitude of platforms - and environment-specific configuration files in sync leads to repetitive configuration work.
```

Comment:
shouldn't it  "platform-specific config" or "platforms and environment-specific"?